packages/react-components/react-infolabel-preview/stories/InfoLabel/InfoLabelDefault.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+
+import { Link } from '@fluentui/react-components';
+import { InfoLabel, InfoLabelProps } from '@fluentui/react-infolabel-preview';
+
+export const Default = (props: Partial<InfoLabelProps>) => (
+  <InfoLabel
+    info={
+      <>
+        This is example information for an InfoButton. <Link href="https://react.fluentui.dev">Learn more</Link>
```

Comment:
```suggestion
        This is example information for an InfoLabel. <Link href="https://react.fluentui.dev">Learn more</Link>
```
Not terribly important to change, since technically InfoButton is also correct